packages/esbuild-plugin/package.json

```diff
@@ -18,14 +18,14 @@
     "@vanilla-extract/integration": "^6.2.0"
   },
   "peerDependencies": {
-    "esbuild": ">=0.17.6"
+    "esbuild": ">=0.19.9"
```

Comment:
This will cause a major version bump, which we don't really want. The plugin works with `>=0.17.6`, so we'll leave it as-is.